src/app/search/search-tips/search-tips.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="search-tips">
+  <h4 i18n>Search tips:</h4>
+  <ul>
+    <li *ngFor="let tip of searchTips" i18n>{{tip}}</li>
```

Comment:
It'll not work, as you're rendering dynamic data here. It's better to get rid of the loop here and just list all of the tips with `i18n` attribute